ResourceNamespaceInfo.json

```diff
@@ -5,31 +5,36 @@
   },
   {
     "Namespace": "AadIam:ExternalUserDirectorySettings",
-    "Description": "Azure AD > External user directory settings"
+    "Description": "Azure AD > External user directory settings",
```

Comment:
One way or the other we should be consistent with capitalization. camelCase is usually used for json, so the original properties should really be renamed to description and namespace. 

Would you please do so?

We may want to coordinate a release with the app to avoid breaking anything. 

Also note that the web app should use a json file from the same branch as it was built from.